Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -52,22 +52,55 @@
 module Homebrew
   module_function
 
-  def audit
-    Homebrew::CLI::Parser.parse do
-      switch      "--strict"
-      switch      "--online"
-      switch      "--new-formula"
-      switch      "--fix"
-      switch      "--display-cop-names"
-      switch      "--display-filename"
+  def audit_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `audit` [<options>] [<formulae>]:
+
+        Check <formulae> for Homebrew coding style violations. This should be
+        run before submitting a new formula.
+
+        If no <formulae> are provided, all of them are checked.
+
+      EOS
+      switch      "--strict",            description: "Run additional style checks, "\
+                                                      "including Rubocop style checks."
```

Comment:
What about:

```ruby
switch      "--strict",
  description: "Run additional style checks, including Rubocop style checks."
```

Reply:
In the current style, one could see all the declared options in a single column (I think its more readable this way), but with above style the description would come in between. Its just a choice and I would defer the final call on this to you @MikeMcQuaid

Reply:
I think it'd be preferable to be able to have the description on a single line, thanks 👍